net/ampr-ripd/Makefile

```diff
@@ -0,0 +1,107 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ampr-ripd
+PKG_VERSION:=2.4.2
+PKG_RELEASE:=1
+PKG_SOURCE_URL:=https://yo2loj.ro/hamprojects
+PKG_HASH:=e4635bd0f88c1f2b0777e948a00d0470aa97254ec4b0b8fd75c79d109995a350
+PKG_MAINTAINER:=Dan Srebnick <k2ie@k2ie.net>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tgz
+PKG_LICENSE:=GPL-2.0-only
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/ampr-ripd
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Routing and Redirection
+	DEPENDS:=+kmod-ipip +ip-tiny
```

Comment:
Since you switched the dependency from `ip-full` to `ip-tiny` as per previous discussion, you can specify it as `+ip` instead, as mentioned before, so whichever version a user has installed out of those two will be fine.